### PR TITLE
add mlp.ipynb and some .py files needed for running mlp

### PR DESCRIPTION
**Pull request by oh-yu**

### CODE REVIEW

**Comment by oh-yu** [comment]
1. what is p?
2. PLEASE DO PYLINT(AS FOR train_model.py and mlp.ipynb)
3. save information including model's state_dict, losses, the number of epochs via torch.save()(as for at train_model.py)
4.  Oprimizer => Optimizer
5. 0.5(dropout ratio) should be at #set free parameters
